src/test/regress/expected/multi_deparse_function.out

```diff
@@ -0,0 +1,311 @@
+(1 row)
+
+SELECT deparse_test($cmd$
+ALTER FUNCTION add RESET ALL
+$cmd$);
+                  deparse_test                  
+------------------------------------------------
+ ALTER FUNCTION function_tests.add() RESET ALL;
+(1 row)
+
+SELECT deparse_test($cmd$
+ALTER FUNCTION add RENAME TO summation
+$cmd$);
+                       deparse_test                       
+----------------------------------------------------------
+ ALTER FUNCTION function_tests.add() RENAME TO summation;
+(1 row)
+
+CREATE ROLE function_role;
+NOTICE:  not propagating CREATE ROLE/USER commands to worker nodes
+HINT:  Connect to worker nodes directly to manually create all necessary users and roles.
+SELECT deparse_test($cmd$
+ALTER FUNCTION add OWNER TO function_role
+$cmd$);
+                        deparse_test                         
+-------------------------------------------------------------
+ ALTER FUNCTION function_tests.add() OWNER TO function_role;
+(1 row)
+
+SELECT deparse_test($cmd$
+ALTER FUNCTION add OWNER TO missing_role
+$cmd$);
+                        deparse_test                        
+------------------------------------------------------------
+ ALTER FUNCTION function_tests.add() OWNER TO missing_role;
+(1 row)
+
+SELECT deparse_test($cmd$
+ALTER FUNCTION add SET SCHEMA public
+$cmd$);
+                      deparse_test                      
+--------------------------------------------------------
+ ALTER FUNCTION function_tests.add() SET SCHEMA public;
+(1 row)
+
+SELECT deparse_test($cmd$
+ALTER FUNCTION add DEPENDS ON EXTENSION citus
+$cmd$);
+                          deparse_test                           
+-----------------------------------------------------------------
+ ALTER FUNCTION function_tests.add() DEPENDS ON EXTENSION citus;
+(1 row)
+
+SELECT deparse_test($cmd$
+DROP FUNCTION IF EXISTS add(int,int);
+$cmd$);
+                                 deparse_test                                 
+------------------------------------------------------------------------------
+ DROP FUNCTION IF EXISTS function_tests.add(pg_catalog.int4,pg_catalog.int4);
+(1 row)
+
+-- Check that an invalid function name is still parsed correctly
+SELECT deparse_test($cmd$
+DROP FUNCTION IF EXISTS missing_function(int, text);
+$cmd$);
+                          deparse_test                           
+-----------------------------------------------------------------
+ DROP FUNCTION IF EXISTS missing_function(pg_catalog.int4,text);
+(1 row)
+
+SELECT deparse_test($cmd$
+DROP FUNCTION IF EXISTS missing_schema.missing_function(int,float);
+$cmd$);
+                                        deparse_test                                         
+---------------------------------------------------------------------------------------------
+ DROP FUNCTION IF EXISTS missing_schema.missing_function(pg_catalog.int4,pg_catalog.float8);
+(1 row)
+
+-- clear objects
+SET client_min_messages TO FATAL; -- suppress cascading objects dropping
```

Comment:
can you add a test with multiple options: `alter function xxx  volatile leakproof  SECURITY DEFINER PARALLEL unsafe;
`